scripts/generate_learning_path_markdown.js

```diff
@@ -57,7 +57,6 @@ const getYouTubeImage = (youTubeCode) => {
 
       let { indexContent, ...indexFrontMatter } = yamlFront.loadFront(fs.readFileSync(indexReadPath, 'utf-8'), { contentKeyName: 'indexContent' })
       indexFrontMatter.image = section.image
-      indexFrontMatter.contributors = await getContributors(relative('..', readPath))
```

Comment:
Is this contributor fetching being totally removed or is it moved somewhere else somehow?

Reply:
That's true we just need contributors on the articles themselves not the index.